app/models/concerns/import_utils.rb

```diff
@@ -13,14 +13,26 @@ module ImportUtils
   end
 
   def enclosure_url(entry)
-    url = entry[:feedburner_orig_enclosure_link] || entry[:enclosure].try(:url)
+    url = entry[:feedburner_orig_enclosure_link] || enclosure_url_from_entry(entry)
     clean_string(url)
   end
 
   def default_story_url(story)
     StoryDistributions::EpisodeDistribution.default_story_url(story)
   end
 
+  def clean_title(str)
+    str = clean_string(str)
+    return str if str.length <= 255
+    truncated = str[0..254]
+
+    e = RuntimeError.new("ImportUtils: String length of #{str.length} exceeds title column length of 255: #{truncated}")
```

Comment:
Does this include the story id?  Seems like that might be helpful for debugging later.